docs/PacketFence_Aerohive_Quick_Install_Guide.asciidoc

```diff
@@ -18,433 +18,264 @@ include::includes/global-attributes.asciidoc[]
 
 About this Guide
 ----------------
-This guide has been created in order to help sales engineers, product managers, or network specialists demonstrate the PacketFence capabilities on-site with an existing or potential customer.  It can also provide guidelines to setup a proof of concept for a potential PacketFence deployment using the Aerohive equipment.
+This guide has been created in order to provide guidelines to deploy a Aerohive Wireless APs and the associated management in a PacketFence context, whether it is with a standalone server, a cluster or PacketFencZEN deployxement.
 
-The instructions are based on version {release_version} of the PacketFence Live Image.
+NOTE: Aerohive products are a bit different compared to the other vendors. They support either a local HiveManager (kind of wireless controller) or a cloud-based HVM. However, the configuration is the same for the local and the cloud-based controller. Note that all the config are made on the HVM and then pushed to the APs. The instructions are based on version {release_version} of the PacketFence.
 
 Assumptions
 -----------
-* You will need a USB key with a capacity of at least 4GB;
-* You will need a laptop or a PC from which the PacketFence Live USB key will run from;
-* You will deploy a VLAN enforcement environment according to the customer specifications;
-* Your customer will provide the network equipment (switch, cables, etc) to interconnect this PoC setup;
-* Aerohive Access Point is loaded with HiveOS with version 6 or later;
-* HiveManager with version 6 or later.
+* You already have a PacketFence server(s) up and running, (standalone, cluster or ZEN)
+  ** If not, please refer to the https://packetfence.org/doc/PacketFence_Administration_Guide.html[PacketFence Administration Guide].
+* The network architecture is in order to give acces to the Aerohive Access Point, and has access to internet
+* The VLANs are defined for registration, isolation and management networks
```

Comment:
IMO you need to keep the assumptions we have in other documentation, like:
PacketFence IP address: 192.168.1.5
Normal VLAN: 1
Registration VLAN: 2
Isolation VLAN: 3
RADIUS Secret: useStrongerSecret
AP IP: 172.16.1.1
And have the screenshot related to this information, like the portal url should be http:://192.168.0.5/AeroHIVE::AP